### DISCUSSION TITLE
Touch effect to existing buttons and other controls

### DISCUSSION THREAD

**Discussion by developer9969** [xamarin/XamarinCommunityToolkit#1235] [Q&A]
Hi
Really love this feature however my lack of understanding on  how to use it properly is causing issue in my existing code base.

Scenario
I have an app with buttons etc.. and the style is for the buttons is in the app.xaml.

1. I have added the touch effect to the style
2. Touch effect works
3. Existing button. Command no longer works
4. Replacing with TouchEffect.Command  works

Does it mean that if I want to apply the touch effect to all app buttons I have to replace the existing command with TouchEffect.Command and TouchEffect.CommandParameters?

Also the PressedBackgroundColor does not seem to do anything.. I will create a dummy project and test there
        <Setter Property="xct:TouchEffect.PressedBackgroundColor" Value="Red" />
        <Setter Property="xct:TouchEffect.NormalBackgroundColor" Value="White" />

many thanks for this

**Comment by AndreiMisiukevich** [comment]
Hi,

PressedBackgroundColor should work. Can you try to apply both NormalBackgroundColor and PressedBackgroundColor?

**Comment by AndreiMisiukevich** [comment]
Existing button. Command no longer works

Is it iOS or Android issue? Can you test on both platforms? Cuz it should work as well
